gym/envs/__init__.py

```diff
@@ -204,89 +204,89 @@
 # 2D
 
 register(
-    id='Reacher-v1',
+    id='Reacher-v2',
     entry_point='gym.envs.mujoco:ReacherEnv',
     max_episode_steps=50,
     reward_threshold=-3.75,
 )
 
 register(
-    id='Pusher-v0',
+    id='Pusher-v1',
```

Comment:
I think these should all be the same version (even if that means some environments skip versions).

This makes it obvious that "all the v2+ mujoco environments use version 1.50" or something similar.